src/choropleth.ts

```diff
@@ -50,12 +50,15 @@ enum GeoLevel {
 }
 
 const unknownColor = "#DADCE0"; //material grey 300
-const path = d3.geoPath();
 
 const alaskaFipsCode: string = "02";
+const northernIrelandFipsCode: string = "N";
```

Comment:
I'm guessing we have to add extra styling for Northern Ireland like we do for alaska?  Can you add a comment?

Reply:
Yes, essentially having different border colour for areas that don't have border with the rest - Norther Ireland is on a separate island, Alaska doesn't connect with the rest of the US on the map. Added a comment in code